src/routes/api/user/login/+server.ts

```diff
@@ -0,0 +1,55 @@
+import { authToken, currentUserId } from '$stores';
```

Comment:
Was machen die server ts files? Ist das die Verbindung zwischen Sveltekit und API?

Reply:
Alles unter `/api` sind unsere API-Routen, die wir auf unserer Client-Seite `fetchen` können. Die `/login/server.ts` können wir beispielsweise überall im Code mit `fetch 'api/users/login'` und der richtigen HTTP-Methode aufrufen. 
Dadurch löst unser Client einen Request an unseren NodeJS-Server aus, welcher dann wiederum im hier definierten Code beispielsweise unser Backend aufruft. Ist ganz cool, weil wir dann auch hier in der `server.ts` uns die Antwort vom Backend so mappen können, dass sie unser Client direkt verwerten kann ohne selbst was dafür tun zu müssen.

Also `Client` - (fetch `api/...`) > `UI Server` - (fetch `expenseapi/...` > `Backend Server`